app/Http/Controllers/Search/MiscController.php

```diff
@@ -329,24 +329,32 @@ public function attributeTemplates(Request $request)
      */
     protected function buildSearchResults(string $term, string $class, array $excludes = [])
     {
+        $name = 'name';
```

Comment:
I feel like this function now does too much, mixing many ideas together. Can you revert it and have a different buildGenres() function that just focuses on genres? Thanks!